### PR TITLE
Get block hashes by timestamp range

### LINKED ISSUE #204: Get blocks by date

### PR DESCRIPTION
**Pull request by pnagurny**
Closes #204

### CODE REVIEW

File: docs/services/db.md [hunk 1]
```diff
@@ -32,3 +32,13 @@ node.getBlock(blockHash, function(err, block) {
   //...
 });
 ```
+
+Get Block Hashes by Timestamp range
+
+var time1 = 1441911000; // Notice time is in seconds not milliseconds
```
Comment (braydonf) [inline docs/services/db.md]: missing backticks for code formatting

**Comment by kleetus** [comment]
LGTM: tests pass locally

**Comment by braydonf** [comment]
If we can store the indexes as binary, e.g. sending a buffer for the key and value, and receiving a buffer the data storage size will be half the size, and faster to write and read.

**Comment by pnagurny** [comment]
Updated to binary encode leveldb key and value.

**Comment by braydonf** [comment]
LGTM

**Comment by braydonf** [comment]
Testing with https://github.com/bitpay/insight-api/pull/368

**Comment by braydonf** [comment]
LGTM

**Pull request merged**